src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -0,0 +1,208 @@
+import _has from 'lodash/has';
+import _isNil from 'lodash/isNil';
+import EventBus from '../lib/EventBus';
+import NavigationLibrary from '../navigationLibrary/NavigationLibrary';
+import RadarTargetCollection from './RadarTargetCollection';
+import { EVENT } from '../constants/eventNames';
+import { DECIMAL_RADIX } from '../constants/globalConstants';
+import { THEME } from '../constants/themes';
+
+export default class ScopeModel {
+    constructor() {
+        this._eventBus = EventBus;
+        this._navigationLibrary = NavigationLibrary;
+        // TODO: Use this!
+        this._sectorCollection = [];
+        this._theme = THEME.DEFAULT;
+
+        this.radarTargetCollection = [];
```

Comment:
- remove empty line above this one
- this should default to `null` since it later becomes a `RadarTargetCollection` instance

Reply:
done